Update harbour-seaprint-de.ts

cnlpete opened the pull request:
Added a few german translations

attah commented:
Thank you for your contribution!
As you may have seen, there is another PR outstanding for German translations.
Have you coordinated? It looks like many of my comments there are not valid here.

cnlpete commented:
Oh crap, i did not see that one :facepalm: I will check your comments and try to merge accordingly ...

cnlpete commented:
I rebased my changes on the changes from #49, @wheelseal Feel free to look at these changes too

attah commented:
Will do, thanks!
I'll poke him elsewhere too - because i don't speak German, although Swedish helps with guessing some things.

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -116,23 +116,23 @@
     <message>
         <source>Consider using a suffix like &quot;/ipp/print&quot;.</source>
-        <translation type="unfinished"></translation>
+        <translation>ERwägen Sie die Verwendung eines Suffixes wie zum Beispiel &quot;/ipp/print&quot;.</translation>
```
Capitilization error?

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -485,27 +485,27 @@
     </message>
     <message>
         <source>Bottom</source>
         <translation>unten</translation>
```
Capitalization?

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -554,33 +554,33 @@
     <message>
         <source>Unnamed supply</source>
-        <translation type="unfinished"></translation>
+        <translation>unbenannter Toner</translation>
```
Capitalization? This can be a lot more than toners (ink, fuser unit etc), supply is the generic name

cnlpete replied:
I left "Toner", as i am unsure of a word for this atm...

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -787,115 +787,115 @@
     <message>
         <source>alternate-roll</source>
-        <translation type="unfinished"></translation>
+        <translation>abwechselnd rollen</translation>
```
Capitalization?

cnlpete replied:
can you give me a hint, what context this is used in?

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -787,115 +787,115 @@
     <message>
         <source>roll</source>
-        <translation type="unfinished"></translation>
+        <translation>rollen</translation>
```
Capitalization?

cnlpete replied:
can you give me a hint, what context this is used in?

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -787,115 +787,115 @@
     <message>
         <source>large capacity</source>
-        <translation type="unfinished"></translation>
+        <translation>hohe Kapazität</translation>
```
Capitalization?

attah commented on translations/harbour-seaprint-de.ts:
```diff
@@ -787,115 +787,115 @@
     <message>
         <source>main roll</source>
-        <translation type="unfinished"></translation>
+        <translation>Hauptrolle</translation>
```
Capitalization?
(more follows, not commenting on all)

attah reviewed (commented):
Looking good.
Some minor comments...

cnlpete commented:
I am unsure of the usage of the "utils" texts, they all appear to be lowercase in the english source texts, what context are they used in? (i.e. "main roll" - i am guessing some kind of paper source selection?)

attah commented:
Right, i'm a bit blind to my own structure sometimes.
All strings but one "Unknown" are in the [ippName](https://github.com/attah/harbour-seaprint/blob/5f4b58884641590ef52148de130ac8a142b55cb2/qml/pages/utils.js#L62)  function. It maps enums and not-so-user-friendly names to translatable user-friendly ones.

Most will appear as choices when selecting options for a print job. But "job-state" and "printer-state" will appear on other pages, jobs and details pages respectively. I kept all choices lowercase, except the formats that are obviously names. If doing them capitalized is more appropriate for German, go right ahead - just be consistent. :)

cnlpete commented:
@attah if you compare my changes against #49 (https://github.com/wheelseal/harbour-seaprint/compare/master...cnlpete:german-translations) you can see that i changed a few Uppercase/lowercase occurrences and also corrected a few typos

I am still unsure of how to translate all the ipp option strings, therefore i did not really touch those...

Even though it might not be at 100%, i think it is still a bit better than having a mixture of english and german texts

attah commented:
Hmm, i see diff for things that were corrected on both PRs... i'll double-check locally.

cnlpete commented:
i rebased my changes on #49, so it includes all the commits from wheelseal. please check the [diff link](https://github.com/wheelseal/harbour-seaprint/compare/master...cnlpete:german-translations) for the changes to wheelseals branch

attah commented:
Thanks, that looks much more like expected. Merging.
